notifier/alerts.go

```diff
@@ -113,3 +120,32 @@ func renderMessage(sourceTemplate string, event models.Event) string {
 	return renderedTemplate.String()
 
 }
+
+// Build HTTP headers based on template
+func renderHeaders(headers []map[string]string, event models.Event) []map[string]string {
```

Comment:
I could suppose, there is still no possibility to use [os.Getenv](https://pkg.go.dev/os/#Getenv) interfaces:

```log
2024/08/12 23:41:23 +0300 WRN ../tmp/src/notifier/alerts.go:134 > Failed to render HTTP header error="template: custom:1: function \"env\" not defined"
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x5bb026]

goroutine 81 [running]:
text/template.errRecover(0xc0001ff520)
	/usr/local/go/src/text/template/exec.go:170 +0x15a
panic({0x874b40?, 0xce20f0?})
	/usr/local/go/src/runtime/panic.go:770 +0x132
text/template.(*Template).execute(0x0, {0xa0c4a0?, 0xc0001de0c0?}, {0x8e19e0?, 0xc0004a8200?})
	/usr/local/go/src/text/template/exec.go:221 +0x1e6
text/template.(*Template).Execute(...)
	/usr/local/go/src/text/template/exec.go:207
github.com/0x2142/frigate-notify/notifier.renderHeaders({_, _, _}, {{0x84c600, 0xc00012ae80}, {0x845bc0, 0xc000136348}, {0xc00015c1c8, 0x13}, {{0x0, ...}, ...}, ...})
	/tmp/src/notifier/alerts.go:138 +0x393
github.com/0x2142/frigate-notify/notifier.SendNtfyPush({{0x84c600, 0xc00012ae80}, {0x845bc0, 0xc000136348}, {0xc00015c1c8, 0x13}, {{0x0, 0x0, 0x0}, {0x0, ...}, ...}, ...}, ...)
	/tmp/src/notifier/nfty.go:68 +0x9c5
created by github.com/0x2142/frigate-notify/notifier.SendAlert in goroutine 11
	/tmp/src/notifier/alerts.go:58 +0x7d3
```

after I've tried

```yaml
alerts:
  ntfy:
    headers:
      - Authorization: Basic {{ env.Getenv "FN_NTFY_AUTH_BASIC" }}
```

Reply:
Just added this in #114 

Should be able to do this now: 
```yaml
alerts:
  ntfy:
    headers:
      - Authorization: Basic {{ env "FN_NTFY_AUTH_BASIC" }}
```

Let me know how that works for you?

Reply:
Just checked with the new build. It works, thanks a lot!